crypto/secp256k1/secp256k1.go

```diff
@@ -179,3 +195,55 @@ func (pubKey PubKeySecp256k1) Equals(other crypto.PubKey) bool {
 	}
 	return false
 }
+
+// VerifyBytes verifies a signature of the form R || S.
+// It rejects signatures which are not in lower-S form.
+func (pubKey PubKeySecp256k1) VerifyBytes(msg []byte, sigStr []byte) bool {
```

Comment:
I think we should perform a test that the current version and previous is compatible, all the function and outputs should be compared.